docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
       ZOOKEEPER_CLIENT_PORT: 2181
       ZOOKEEPER_SERVER_ID: 1
       ZOOKEEPER_SERVERS: zoo1:2888:3888
+    labels:
+      com.centurylinklabs.watchtower.enable: "false"
```

Comment:
watchtower by default, watches all running docker containers. We do not want it to update services we do not control, which includes mongo, zookeeper and kafka, so that they may continue to run in their current stable versions, thus "false".